### ISSUE TITLE
Identify good candidate counties for scraping

### ISSUE THREAD

**Issue by normaljosh** [open-austin/azure-indigent-defense#41]
It seems that court records differ from county to county. We could use help with going through the counties at https://github.com/open-austin/Odyssey-Court-Records-to-JSON/blob/main/resources/texas_county_data.csv , and getting this information: 

1. Is there a calendar search function? 
2. Try to open a specific case that is set as a Jury Trial
3. Save a screenshot of the case
4. Update the spreadsheet to indicate a screenshot has been taken

Example of a jury trial:
![image](https://user-images.githubusercontent.com/6571497/190292085-13bbc475-4df6-4b66-bf94-a0acf65dd347.png)

**Comment by tpadmanabhan** [comment]
Validate with Nick

**Comment by tpadmanabhan** [comment]
per @nicolassaw this info should be included in the referenced csv

**Comment by tpadmanabhan** [comment]
Linked to #58

**Comment by nicolassaw** [comment]
I met with Nate to discuss the project and what counties we should select next for development. In short, it makes sense to identify counties with public defender's offices and Odyssey, with bonus points if they are close by. This will allow us to compare the differences in practice between institutional public defenders and private appointed counsel.

    Background: The original goal of the project was first to find out whether there is a difference in motions practice between different types of defense attorneys. It was hypothesized that full time public defenders file more motions (and litigate more) than court-appointed private attorneys and comparable to (or even more than) private counsel. And this results in better case dispositions for client's represented by public defenders. 

    Counties List: The Texas Indigent Defense Commission publishes a map of Texas with all counties with public defender's offices across Texas: https://tidc.texas.gov/media/txcfktwt/managed-indigent-defense-systems-chart-as-of-nov-2023-final.pdf. It is also important to consider close proximity to Austin for validating online records in person: Hays Count is close by and has Odyssey.

    Next Steps: I will work on adding a new column to the Excel sheet that indicates whether the county has a public defender's office. If it does have a PDO, I will then update its link and whether it has Odyssey to create a short list of potential counties to develop.

**Comment by nicolassaw** [comment]
I've updated the CSV of county data to include a column for whether there is a PDO, I've updated a few other Odyssey links, and I've added notes for how far certain places are from Austin to try and identify places within 3.5 hours drive (at max). The date the Odyssey link was updated is also a column on the table.

Based on discussion with Nate, I think these may be a few good candidates for development and why:

- **Hays:** 241,067 population. 1 hour drive from Austin. Has 2003 Odyssey. Has a PDO run by Neighborhood Defender Service (NDS) that takes up to 25% of cases.
- **Brazos:** 233,849 population. 2 hours drive from Austin. Has a PDO.
- **Fort Bend:** 822,779 population. 2.5 hours drive from Austin. Has 2003 Odyssey. Has a PDO for 10% of cases.
- **Nueces:** 353,178 population. 3.25 hours drive from Austin. Has 2003 Odyssey. Has a mental health PDO taking no more than 15% of cases.
- **Galveston:** 350,682 population. 3.5 hours drive from Austin. Has 2017 Odyssey. Has a mental health PDO.

These two don't have PDOs but do have some high attorney caseloads:

- **Bell:** 370,647 population. 1.25 hours drive from Austin. Has 2017 Odyssey. Does not have a PDO but has 4th highest number of attorneys taking lots of cases (132).
- **Williamson:** 609,017 population. .75 hours drive from Austin. Has 2003 Odyssey. Does not have a PDO but has 44 attorneys with high case loads.
